T20847 - Add docs about customize the payment methods in the PaymentCreatorActivity (#132)
omise/omise-android: README.md

## Usage

> ### Credit Card activity

Reviewer: At some point, we should rewrite this so that it refers to cards, not just credit cards.  I suppose we have to explain it carefully as we do have an class called `CreditCardActivity`.

Author: @danfowler Do you some references about the cards in our documents?

method.

### Payment Creator activity

> Another way to use the Omise Android SDK is to integrate the `PaymentCreatorActivity` in 

Reviewer: ```suggestion
Another way to use the Omise Android SDK is to integrate the `PaymentCreatorActivity`
```

> order to allow users to create a payment source from the list of sources available for the implementer.

Reviewer: ```suggestion
to allow users to create a payment source from the list of sources available for the account.
```

    intent.putExtra(OmiseActivity.EXTRA_AMOUNT, 150000L)
    intent.putExtra(OmiseActivity.EXTRA_CURRENCY, "thb")

>     // you can retrieve your account's capabilities through the SDK (will be explained in the below)

Reviewer 2: Isn't here something missing? I think it should be `will be explained in the section below` or `will be explained below`.

Author: @tobias90 Thanks, I has updated in this commit [bc0b5ac](https://github.com/omise/omise-android/pull/132/commits/bc0b5ac40517334d66776bb68008c7ccd335e533)

Replace the string `pkey_test_123` with the public key obtained from your Omise dashboard.

> Declare `capability` variable as a `Capability` put the `capability` with `OmiseActivity.EXTRA_CAPABILITY` key to the `Intent`. So, the `PaymentCreatorActivity` will display the payment methods that contains in the object. 

Reviewer: ```suggestion
Declare a `capability` variable as a `Capability` object and pass it as the value for the `OmiseActivity.EXTRA_CAPABILITY` key for your `Intent`. This way, the `PaymentCreatorActivity` will display the payment methods contained in the `Capability` object. 
```

There are 2 options to retrieve the Capability object. 

1. You can retrieve the Capability object from your account's capabilities through the [Capability](#retrieve-capabilities). 

> 2. Or you can create a `Capability` object to create your own capaiblities by using the helper function `Capability.create()`. 

Reviewer: ```suggestion
2. Or you can create a `Capability` object to create your own capabilities using the helper function `Capability.create()`. 
```

    **Here is the sample:**

    ```kotlin
    val capability = Capability.create(
            allowCreditCard = true,
            sourceTypes = listOf(SourceType.PromptPay, SourceType.TrueMoney)
    )
    ```

    > **Note**

>     > Ensure you enter the supported payment methods with your account's capabilities. If not it won't able to create a source to continue the payment process.

Reviewer: ```suggestion
    > Ensure you are adding payment methods supported by the account. 
    > If not, you won't be able to create a source to continue the payment process.
```
